cmd/nitro/init.go

```diff
@@ -812,21 +789,22 @@ func openInitializeExecutionDB(ctx context.Context, stack *node.Node, config *No
 		testUpdateTxIndex(executionDB, chainConfig, &txIndexWg)
 	} else {
 		var initDataReaderHasAccounts bool
+		// TODO: Should this be part of getNewBlockchain()?
```

Comment:
Nope, this is only used for validateGenesisAssertion, and validateGenesisAssertion shouldn't be part of getNewBlockchain.